docs/source/docs/hardware/customhardware.md

```diff
@@ -8,29 +8,54 @@ By default, PhotonVision attempts to make minimal assumptions of the hardware it
 
 ## LED Support
 
-For Raspberry-Pi based hardware, PhotonVision can use [PiGPIO](https://abyz.me.uk/rpi/pigpio/) to control IO pins. The mapping of which pins control which LED's is part of the hardware config. The pins are active-high: set high when LED's are commanded on, and set low when commanded off.
+When running on Linux, PhotonVision can use [diozero](https://www.diozero.com) to control IO pins. The mapping of which pins control which LED's is part of the hardware config. The illumination LED pins are active-high: set high when LED's are commanded on, and set low when commanded off.
```

Comment:
Are there any nuances here about GPIO pin numbering? Back in the rpi days we had physical pin numbers != gpio pin number != broadcomm pin number, how is this managed with gpiod?

Reply:
Cool! Should our docs link directly to a diagram for raspberry pi and orange pi? I was able to find it via the diozero docs, but maybe bringing it more visible would be good

Reply:
Orange Pi hasn't kept the pin numbering consistent between different versions. I am hesitant to include diagrams for Orange Pi due to this, but linking to external pages for the popular SBCs might be reasonable.

Additionally diozero enumerates all available GPIO chips when it initializes, with the pin numbers being calculated by adding the chip's line offset to the pin's line number. For RPi hardware this is simple since all pins on the main header are part of chip 0. It seems like this may not be the case for Orange Pi and Rubik Pi though.

Reply:
Besides the Orange Pi 5, are there any other models that are popular enough to track down resources for?

Reply:
Cool. Can we add a diagram for / links for specifically raspberry pi for now?